keps/sig-network/1672-tracking-terminating-endpoints/README.md

```diff
@@ -148,6 +148,287 @@ of the [EndpointSlice API work](/keps/sig-network/20190603-endpointslices/README
 Since this is an addition to the EndpointSlice API, the version skew strategy will follow that
 of the [EndpointSlice API work](/keps/sig-network/20190603-endpointslices/README.md).
 
+## Production Readiness Review Questionnaire
```

Comment:
Will this be filled out AND REVIEWED before EOD tomorrow?

Reply:
Yeah, I'm working on it now, just pushing what I have so far